cocotb/drivers/ad9361.py

```diff
@@ -48,13 +48,37 @@ def _rx_clock(self):
 
     def send_data(self, i_data, q_data, i_data2=None, q_data2=None,
                   binaryRepresentation=BinaryRepresentation.TWOS_COMPLEMENT):
+        '''

```

Comment:
While your updating docstrings, the standard, per [PEP-257](https://www.python.org/dev/peps/pep-0257/), indicates that docstrings should always use triple double quotes `"""` (as opposed to triple single quotes `'''`).